.github/workflows/ci.yml

```diff
@@ -59,6 +59,8 @@ jobs:
         run: |-
           wget https://github.com/OmniBlade/binutils-gdb/releases/download/omf-build/omftools.tar.gz
```

Comment:
can you change this so we only extract `jwasm` (and same for Dockerfile if appropriate)?